app/views/settings/mastodon_authentication.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :subnav do %>
+  <a href="/settings">Back to Settings</a>
+<% end %>
+
+<div class="box wide">
+  <%= form_with url: "/settings/mastodon_auth", method: :get do |f| %>
```

Comment:
Use the Rails url generation here and above. Run `rails routes` in your console to see the name for this endpoint, then add `_path` to generate a relative URL.